examples/monitoring-alert/main.tf

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "random_string" "secret-suffix" {
+  length  = 4
+  special = false
+  upper   = false
+}
+
+module "secret-manager" {
```

Comment:
i realized that cmek isn't enabled. plz consider for a future PR

Reply:
Sure, the module I'm using accepts passing a kms key for encryption. In a next PR, I'll add cmek to the example and add the checks in the test.